### PR TITLE
Address bug in S3 uploading logic

### PR DESCRIPTION
**Pull request by michplunkett**
## Related PRs
https://github.com/lucyparsons/OpenOversight/pull/1020

## Description of Changes
The logic of a badly formed image file was being coupled with a non-allowed file type, that's been fixed.

Here's the `415.tiff` image:

## Tests and linting
 - [x] This branch is up-to-date with the `develop` branch.
 - [x] `pytest` passes on my local development environment.
 - [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

File: OpenOversight/tests/conftest.py [hunk 1]
```diff
@@ -330,7 +330,7 @@ def session(db):
 
 
 @pytest.fixture
-def test_png_BytesIO():
+def test_png_bytes_io():
```
Comment (michplunkett) [inline OpenOversight/tests/conftest.py:333]: Python functions should be in all lower case.

File: OpenOversight/tests/test_utils.py [hunk 2]
```diff
@@ -215,89 +214,88 @@ def test_unit_choices(mockdata):
...
             filename = upload.filepath.split("/")[-1]
             filename_parts = filename.split(".")
             assert len(filename_parts) == 2
 
 
-def test_upload_image_to_s3_and_store_in_db_throws_exception_for_unrecognized_format(
-    mockdata, client
+def test_save_image_to_s3_and_db_invalid_image(mockdata, client):
+    with pytest.raises(ValueError):
+        save_image_to_s3_and_db(BytesIO(b"invalid-image"), 1, 1)
```
Comment (michplunkett) [inline OpenOversight/tests/test_utils.py:266]: New test.

File: OpenOversight/tests/test_utils.py [hunk 3]
```diff
@@ -215,89 +214,88 @@ def test_unit_choices(mockdata):
...
+
+
+def test_save_image_to_s3_and_db_unrecognized_format(
+    mockdata, test_tiff_bytes_io, client
 ):
-    with pytest.raises(PIL.UnidentifiedImageError):
-        upload_image_to_s3_and_store_in_db(BytesIO(b"invalid-image"), 1, 1)
+    with pytest.raises(ValueError):
+        save_image_to_s3_and_db(test_tiff_bytes_io, 1, 1)
```
Comment (michplunkett) [inline OpenOversight/tests/test_utils.py:273]: Use `tiff` image which we don't accept.

**Review by b-meson — APPROVED** [review]
Reviewed it twice and looks great to me.

**Pull request merged**